src/proxy-ui-api/src/main/resources/openapi-definition.yaml

```diff
@@ -4489,13 +4489,39 @@ paths:
           description: resource requested does not exists
         '500':
           description: internal server error
+  /system/node-type:
+    get:
+      tags:
+        - system
+      summary: get the node type
+      operationId: getNodeType
+      description: <h3>Administrator views the node type</h3>
+      responses:
+        '200':
+          description: node type information
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/NodeTypeResponse'
+        '400':
```

Comment:
How the user-provided parameters could be wrong now in the parameterless endpoint?   If not possible, I think this 400 should not be listed here neither.
Is there possibly some special auth or X-Road headers that could be in wrong format?

Reply:
I removed all error responses that are not possible. Left `401`, `403` and `500` since those are at least theoretically possible.